tests/integration_tests/performance/test_block_performance.py

```diff
@@ -393,3 +395,81 @@ def test_block_vhost_user_performance(
         fio_block_size,
         vm,
     )
+
+
+@pytest.mark.nonci
+@pytest.mark.parametrize("vcpu_count", [1, 2], ids=["1vcpu", "2vcpu"])
+def test_vhost_user_block(microvm_factory, guest_kernel, rootfs, vcpu_count, metrics):
+    """
+    This test simply tries to boot a VM with
```

Comment:
this is no longer the purpose of the test, right?

Reply:
updated the comment.